Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2866,10 +2872,36 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
 }
 
 #if PLATFORM(MAC)
+void UnifiedPDFPlugin::accessibilityScrollToPage(int pageIndex)
+{
+    scrollToPage(pageIndex);
+}
+
+FloatRect UnifiedPDFPlugin::convertFromPDFPageToScreenForAccessibility(const FloatRect& rect, PDFDocumentLayout::PageIndex pageIndex) const
```

Comment:
Consider a different name for the `rect` argument, one which reflects which coordinate space it's in?